syntax = "proto2";
import "game_event.proto";

message Event_SetCardCounter {
    extend GameEvent {
        optional Event_SetCardCounter ext = 2015;
    }
    optional string zone_name = 1;
    optional sint32 card_id = 2;
    optional sint32 counter_id = 3;
    optional sint32 counter_value = 4;
}
